services.yaml

```diff
@@ -269,6 +269,28 @@ paths:
           $ref: "#/responses/OK"
         401:
           $ref: "#/responses/UnauthorizedError"
+  /services/{env_id}/team/change-role:
+    post:
+      tags:
+        - Team Management
+      summary: "Edit roles"
```

Comment:
if the URL is change-role, why the summary is "edit roles"? i would stick to one verb (not blocking)